_includes/referendum_measure_heading.html

```diff
@@ -0,0 +1,8 @@
+{% case include.referendum_measure_display -%}
+{% when 'referendum' %}
+Referendums
+{% when 'measure' %}
+Ballot measures
+{% when 'proposition' %}
+Propositions
```

Comment:
Not sure I understand the difference between referendum, measure, and proposition.